scripts/34-Plain-Text-Team.md

```diff
@@ -0,0 +1,601 @@
+
+What are programmers using, and can non-programmers use it too?
+
+---
+
+# Enter Git
+
+- GitHub
+- GitLab
+- Bitbucket
+- SourceForge
+- Etc.
+
+notes:
+
+The answer is yes, yes we can.
+
+I recommend you use the most popular distributed version control system on the planet: Git.
+
+You'll use this through one of the many git web hosts, the largest of which is GitHub, which I recommend for most people.
+
+---
+
+# Popularity Matters
+
+notes:
+
+Though I mention GitHub primarily in this video, I'm not sponsored by them, I just acknowledge that popularity matters.
+Support, experience, and integrations with other services will all be far, far easier if you use the standard.
+
+All these tools started as a web interface around the incredible tool: Git.
+
+---
+
+## Aside:
+
+# Linux & Git
+
+notes:
+By the way, the creator of Linux, Linus Torvalds, also later created git, to solve the problem that he created: that the Linux project had become SO LARGE that existing plain text collaboration tools were not scaling.
+
+He jokes that he named his first project, Linux, after himself, and so it was natural to name the second one after himself too!
+
+---
+
+# Github Et Al. Are Greater Than the Sum of Their Parts
+
+notes:
+From simple code-hosting beginnings, these git services have grown to be so much more than that, trusted by the largest projects in the world, built by the largest companies in the world.
+
+The foundation of my ideal dev team uses the raw materials that GitHub has given us.
+
+What are the raw materials?
+
+I'll show you this with a demo:
+We're going to build a GitHub organisation for No Boilerplate.
+
+This video is not sponsored by GitHub, my work is possible, thanks to viewers like you.
+
+---
+
+![[nb-patreon-aug-23.png|700]]
+
+https://www.patreon.com/noboilerplate
+
+notes:
+
+If you'd like to see and give feedback on my videos up to a week early, as well as get discord perks, and even your name in the credits, it would be very kind of you to check my Patreon.
+
+I'm also offering a limited number of mentoring slots. If you'd like 1:1 tuition on Rust, Python, Web tech, Personal organisation, or anything that I talk about in my videos, do sign up and let's chat!
+
+It's just me running this channel, and I'm so grateful to everyone for supporting me on this wild adventure.
+
+Let's make our plain text team:
+
+---
+![[repo.png]]
+
+# Repos
+
+notes:
+The foundational unit with any git host is the repo.
+This doesn't just correspond with one git repository, but one logical project or subproject.
+Organisational tools like the Wiki (for documentation), Projects (for project management) and more can sit here, right next to your project's files, right where you need them.
+
+---
+
+![[wiki2.png]]
+
+## Wikis
+
+notes:
+Each GitHub repository has a wiki, a folder of linked markdown files that anyone with access can edit, either in the friendly web editor, or, using git, on their own computer with whatever editor they like.
```

Comment:
Given how this presentation focuses on how plaintext frees you from being tied into proprietary solutions, I think it's work mentioning explicitly that GitHub wikis [can be cloned](https://gist.github.com/subfuzion/0d3f19c4f780a7d75ba2) just like a normal repo. I think this isn't necessarily obvious even to people who have used GitHub wikis quite a lot - I didn't actually know this until I googled it.

Reply:
very good point, I love that feature

Reply:
I actually did say that you can use git with wikis, but I'll change the language to be clearer